docs/DeveloperGuide.md

```diff
@@ -192,9 +192,88 @@ This section describes some noteworthy details on how certain features are imple
 
 ##### Implementation
 
+
+The mechanism for adding logs is facilitated by the 'ByIndexByNameCommand', 'AddLogCommand', 'AddLogCommandParser' in the `Logic` component,
+`UniqueLogList` and `Person` classes in the `Model` component, and `JsonAdaptedLog` in the `Storage` component.
+In particular:
+1. In the model, `Person` now has an additional `UniqueLogList` field encapsulating some number
+   of `Log` objects.
+
+![](images/LogFeaturesModelClassDiagram.png)
```

Comment:
Do give the images a name (inside the square brackets) so they can be referenced down the line if need be. This applies for other images below as well.